envoy/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Envoy container
 
 # Stage1: build from source
-FROM quay.io/cybozu/golang:1.20-focal AS build
+FROM quay.io/cybozu/golang:1.20-jammy AS build
```

Comment:
How about using Go 1.21?

Reply:
When I tried to use Go 1.21, CI failed because circleci job's version is 1.20.
I'm not sure I can update base Go version in this time.
So I don't use newer version Go.

This is error log.
```
make: Entering directory '/work/envoy'
go mod tidy
go: go.mod file indicates go 1.21, but maximum version supported by tidy is 1.20
make: *** [Makefile:9: check-generate] Error 1
make: Leaving directory '/work/envoy'
```